lib/spree/graphql/lazy_resolvers.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Spree::GraphQL::LazyResolvers
+  mattr_accessor :_lazy_resolvers, instance_accessor: false
+
+  self._lazy_resolvers = {}
```

Comment:
You can use directly the `default: ` argument on [`mattr_accessor`](https://github.com/rails/rails/pull/29294).

Reply:
Thank you!